### PR TITLE
New version: CPUID.CPU-Z version 2.09

### LINKED ISSUE #135668: [Update Request]: Update CPUID.CPU-Z to 2.09
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

CPUID.CPU-Z

### Package Version

2.0.9

### Please describe the changes you would like to see

2.0.9 was released:

> * Improved support of Intel Meteor Lake and preliminary support of Arrow Lake.
> *  AMD Hawk Point and Hawk Point 2 (Zen 4/Zen 4c).
> *  NVIDIA RTX 4070 SUPER (AD104-350).
>  * The benchmark can now be ran on a single coreset/cluster.

https://www.cpuid.com/softwares/cpu-z.html#version-history

### PR DESCRIPTION
**Pull request by Exorcism0666**
### Pull request has been created with [Komac](https://github.com/russellbanks/Komac) v1.11.0 :rocket:

- Resolve: #135668

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135671)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-01-21T20:02:34.7862917Z ##[error] Manifest Error: Duplicate installer entry found.

(Automated response - build 626.)

File: manifests/c/CPUID/CPU-Z/2.09/CPUID.CPU-Z.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  InstallerUrl: https://download.cpuid.com/cpu-z/cpu-z_2.09-cn.exe
+  InstallerSha256: FC7F73DC51FBC999593109EBCC0B4AF2DFE3540DCCE8D37FF83AC8BEBEFB2E2D
+- InstallerLocale: zh-SG
+  Architecture: x64
+  InstallerUrl: https://download.cpuid.com/cpu-z/cpu-z_2.09-cn.exe
+  InstallerSha256: FC7F73DC51FBC999593109EBCC0B4AF2DFE3540DCCE8D37FF83AC8BEBEFB2E2D
+- InstallerLocale: zh-SG
+  Architecture: x86
+  InstallerUrl: https://download.cpuid.com/cpu-z/cpu-z_2.09-cn.exe
+  InstallerSha256: FC7F73DC51FBC999593109EBCC0B4AF2DFE3540DCCE8D37FF83AC8BEBEFB2E2D
```
Comment (stephengillie) [inline manifests/c/CPUID/CPU-Z/2.09/CPUID.CPU-Z.installer.yaml:61]:
```suggestion
```

**Comment by stephengillie** [comment]
> Specified hash doesn't match.
Actual hash:   C9CD0151FCD74A0F301CD4C5832AB67BD38D08C7AE1FAF39463FE380C2766C6A
Expected hash: fc7f73dc51fbc999593109ebcc0b4af2dfe3540dcce8d37ff83ac8bebefb2e2d
Specified hash doesn't match.
Actual hash:   657B7F4E403269768CD20F1A5B481878CECC775522CEEF119B2723E0844F2361
Expected hash: 92083bd65d594be47759cd196e3a27e75a3e13a00d76460f117dbb38bedb755f

File: manifests/c/CPUID/CPU-Z/2.09/CPUID.CPU-Z.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  InstallerUrl: https://download.cpuid.com/cpu-z/cpu-z_2.09-en.exe
+  InstallerSha256: 92083BD65D594BE47759CD196E3A27E75A3E13A00D76460F117DBB38BEDB755F
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerUrl: https://download.cpuid.com/cpu-z/cpu-z_2.09-en.exe
+  InstallerSha256: 92083BD65D594BE47759CD196E3A27E75A3E13A00D76460F117DBB38BEDB755F
+- InstallerLocale: zh-SG
+  Architecture: x64
+  InstallerUrl: https://download.cpuid.com/cpu-z/cpu-z_2.09-cn.exe
+  InstallerSha256: FC7F73DC51FBC999593109EBCC0B4AF2DFE3540DCCE8D37FF83AC8BEBEFB2E2D
```
Comment (stephengillie) [inline manifests/c/CPUID/CPU-Z/2.09/CPUID.CPU-Z.installer.yaml:25]:
```suggestion
  InstallerSha256: C9CD0151FCD74A0F301CD4C5832AB67BD38D08C7AE1FAF39463FE380C2766C6A
```

File: manifests/c/CPUID/CPU-Z/2.09/CPUID.CPU-Z.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- InstallerLocale: zh-SG
+  Architecture: x86
+  InstallerUrl: https://download.cpuid.com/cpu-z/cpu-z_2.09-cn.exe
+  InstallerSha256: FC7F73DC51FBC999593109EBCC0B4AF2DFE3540DCCE8D37FF83AC8BEBEFB2E2D
```
Comment (stephengillie) [inline manifests/c/CPUID/CPU-Z/2.09/CPUID.CPU-Z.installer.yaml:29]:
```suggestion
  InstallerSha256: C9CD0151FCD74A0F301CD4C5832AB67BD38D08C7AE1FAF39463FE380C2766C6A
```

File: manifests/c/CPUID/CPU-Z/2.09/CPUID.CPU-Z.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerUrl: https://download.cpuid.com/cpu-z/cpu-z_2.09-en.exe
+  InstallerSha256: 92083BD65D594BE47759CD196E3A27E75A3E13A00D76460F117DBB38BEDB755F
```
Comment (stephengillie) [inline manifests/c/CPUID/CPU-Z/2.09/CPUID.CPU-Z.installer.yaml:17]:
```suggestion
  InstallerSha256: 657B7F4E403269768CD20F1A5B481878CECC775522CEEF119B2723E0844F2361
```

File: manifests/c/CPUID/CPU-Z/2.09/CPUID.CPU-Z.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerUrl: https://download.cpuid.com/cpu-z/cpu-z_2.09-en.exe
+  InstallerSha256: 92083BD65D594BE47759CD196E3A27E75A3E13A00D76460F117DBB38BEDB755F
```
Comment (stephengillie) [inline manifests/c/CPUID/CPU-Z/2.09/CPUID.CPU-Z.installer.yaml:21]:
```suggestion
  InstallerSha256: 657B7F4E403269768CD20F1A5B481878CECC775522CEEF119B2723E0844F2361
```

**Review by stephengillie — APPROVED** [review]

**Comment by MagicAndre1981** [comment]
>  don't hesitate to ping @Exorcism0666, I'll get a notification.

works fine, just got the update 👍

**Pull request merged**